roc/stringer_test.go

```diff
@@ -0,0 +1,62 @@
+package roc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStringer(t *testing.T) {
+	for i := -10; i <= 1000; i++ {
+		assert.NotEmpty(
+			t,
+			Interface(i).String(),
+			"Interface String func must always return something",
```

Comment:
I think these messages:

```
"Interface String func must always return something"
```

are redundant, because they basically repeat what assert.NotEmpty means. I suggest to remove them.